### PR TITLE
Merge changelogs list

### PR DESCRIPTION
**Pull request by kwankyu**
I mage the merge with the changelogs, omitting some wrong entries.

Then I removed a few obvious duplicates, but I guess there are many new duplicates.

At the end of the list, there are some names which look like trac account names. They may be removed. But I did not.

### CODE REVIEW

File: conf/contributors.xml [hunk 1]
```diff
@@ -2622,7 +3430,11 @@ Please keep the list in alphabetical order by last name!
  location="Universität Würzburg, Germany"
  url="http://www.mathematik.uni-wuerzburg.de/~mueller"
  trac="pmueller"/>
- github="ypfmde"/>
+ github="ypfmde"/&gt;
```
Comment (mkoeppe) [inline conf/contributors.xml:3433]: this looks like a typo

Comment (mkoeppe) [reply conf/contributors.xml:3433]: previously line has /> too much

File: conf/contributors.xml [hunk 2]
```diff
@@ -2632,10 +3444,12 @@ Please keep the list in alphabetical order by last name!
  name="Gregg Musiker"
  location="University of Minnesota"
  description="Maple interface; documentation; design"/>
- trac="gmoose05"/>
+ trac="gmoose05"/&gt;
```
Comment (mkoeppe) [inline conf/contributors.xml:3447]: same here

**Review by mkoeppe — APPROVED** [review]

**Comment by mkoeppe** [comment]
> What should we do about these names?

I think they should become `<contributor trac="..."/>`

**Comment by haraldschilly** [comment]
@kwankyu I merged this, but then I discovered xml is broken. e.g. `github="ypfmde"/>;` ends with a semicolon, that's not ok. I'll fix it.

**Comment by kwankyu** [comment]
> @kwankyu I merged this, but then I discovered xml is broken. e.g. `github="ypfmde"/>;` ends with a semicolon, that's not ok. I'll fix it.

Right. Thanks!

**Comment by kwankyu** [comment]
> > What should we do about these names?
> 
> I think they should become `<contributor trac="..."/>`

Okay. I will do that, and do the fix in the comment

https://github.com/sagemath/website/pull/346#issuecomment-1280443514

in a new PR.

**Pull request merged**